jenkins_wrapper.sh

```diff
@@ -123,9 +123,10 @@ esac
     OPTS+=('-b')
   fi
 
-  ./bin/deploy "${OPTS[@]}"
+  ./bin/deploy -r "$LSST_SPLENV_REF" "${OPTS[@]}"
```

Comment:
What happens if someone deletes the ref from the jenkins submission form?

Reply:
https://github.com/lsst-sqre/ci-scripts/blob/master/jenkins_wrapper.sh#L41 kicks in, I believe.